gramineproject/gsc
# [Docs] Fix RTD integration, remove Doxygen

## Description of the changes 

- `Documentation/requirements.txt` weren't found by RTD, so we need to point to them explicitly in `.readthedocs.yaml`
- Doxygen was unused, so I removed it.

## How to test this PR? 

`make -C Documentation html && firefox Documentation/_build/html/index.html`

*After* merge, the documentation should be available at https://gramine-gsc.rtfd.io.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gsc/20)
